cmd/internal/cli/build_linux.go

```diff
@@ -186,6 +191,17 @@ func run(cmd *cobra.Command, args []string) {
 			sylog.Fatalf("While performing build: %v", err)
 		}
 	} else {
+		// ensure passphrase or key was supplied with encrypt option
+		if encrypt && !cmd.Flags().Lookup("encryption-key").Changed {
+			sylog.Fatalf("Unable to encrypt container, no passphrase or key path specified.")
+		}
+
+		// ensure we do not build an encrypted container if encrypt option is not specified
+		if !encrypt && cmd.Flags().Lookup("encryption-key").Changed {
+			sylog.Warningf("Encryption key environment variable found, but -e was not specified. NOT encrypting container.")
+			encryptionKey = ""
```

Comment:
Where is this key used?

Reply:
It is in the build config below as it was before this change. This is just preventing an encrypted container build from happening if `-e` isnt there